.github/ISSUE_TEMPLATE/enhancement.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Enhancement
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+# Description of the feature
```

Comment:
I dislike the inconsistency:
- template is called "Enhancement"
- the "about" shown says:  Suggest an idea 
- This says: feature